src/ert/config/gen_kw_config.py

```diff
@@ -146,6 +137,13 @@ def from_config_list(cls, gen_kw: List[str]) -> Self:
                 if item.strip():  # only lines with content
                     transfer_function_definitions.append(item)
 
+        if gen_kw_key == "PRED" and update_parameter:
```

Comment:
Is this perhaps a bit clearer?

"The 'PRED' keyword is no longer automatically excluded from updates. "
        "Set 'UPDATE: FALSE' to exclude this from updates.",